components/automate-deployment/pkg/airgap/bundle_creator.go

```diff
@@ -620,7 +620,7 @@ func (creator *InstallBundleCreator) createTar(InstallBundleCreatorProgress) err
 		return errors.Wrap(err, "Could not write archive")
 	}
 
-	return err
+	return nil
```

Comment:
If you expand the context right above this, L619-621 has just caused a return if err was not nil, so here we know err is nil, and best practice suggested is to be more up front about it.